### ISSUE TITLE
where have the detail process information of achieving cityscapes SoTA?

### ISSUE THREAD

**Issue by dongzhuoyao** [mapillary/inplace_abn#5]

**Comment by pkontschieder** [comment]
@dongzhuoyao 
The currently released code can only deal with image classification tasks (like ImageNet), but we have successfully used InPlace-ABN with a DeepLab3 segmentation head that was trained on top of the WideResNet38 model above. The training settings mostly follows the description in our [paper](https://arxiv.org/abs/1712.02616) but we strive to release code later this year.

**Comment by PkuRainBow** [comment]
@pkontschieder Hi, I have tried to train the WideResNet38-FCN models but find that the performance is even worse than the ResNet-101-FCN model under the same training settings.

For example, on the validation of Cityscapes, their performance is 
_ResNet-101-FCN            75.90
WideResNet38-FCN       73.50_

Thus  I am wondering some details of your training:


**Q1**
Is the **WiderResNetA2** the model that you use?


**Q2**
If the answer to Q1 is yes, then I find that you use two dropout in your **WiderResNetA2**
```
                if mod_id == 4:
                    drop = partial(nn.Dropout2d, p=0.3)
                elif mod_id == 5:
                    drop = partial(nn.Dropout2d, p=0.5)
```
I am wondering do we need to remove these two dropout in our training.

**Q3**
Do you keep the _bn_out_ after the _self.mod7_?

Here is my modification of that snipts,

```
    def forward(self, img):
        tuple_features = list()
        out = self.mod1(img)
        out = self.mod2(self.pool2(out))
        out = self.mod3(self.pool3(out))
        out = self.mod4(out)
        tuple_features.append(out)
        out = self.mod5(out)
        tuple_features.append(out)
        out = self.mod6(out)
        tuple_features.append(out)
        out = self.mod7(out)
        out = self.bn_out(out)
        tuple_features.append(out)
        return tuple_features

```
